src/ext.rs

```diff
@@ -0,0 +1,177 @@
+use std::collections::HashMap;
+
+use alloy_primitives::{Address, B256, U256};
+use revm::{
+    primitives::{Account, AccountInfo, Bytecode, EvmStorageSlot},
+    Database, DatabaseCommit,
+};
+
+/// Extension trait for [`revm::Evm`] with convenience functions for reading
+/// and modifying state.
+pub trait EvmExtUnchecked<Db: Database> {
```

Comment:
Very bikesheddy, but I tend to prefer for ext traits to end with `Ext`

Reply:
yeah, I decided to match the convention of the methods we already have